oraclizeAPI_0.4.25.sol

```diff
@@ -303,23 +353,27 @@ contract usingOraclize {
     OraclizeAddrResolverI OAR;
```

Comment:
Presumably legacy from the eth classic split where `id 2` was both eth classic's testnet id as well as eth's morden testnet?

Not sure if comment needed since this has never been brought up before and the curious could do their own research?

Reply:
this was back when morden was the sole testnet, so it waas referred to testnet generally, just as mainnet had several names along the lines of frontier, homestead etc... etc..